common/persistence/nosql/nosqlplugin/cassandra/plugin.go

```diff
@@ -138,5 +147,21 @@ func toGoCqlConfig(cfg *config.NoSQL) (gocql.ClusterConfig, error) {
 		SerialConsistency:     serialConsistency,
 		Timeout:               cfg.Timeout,
 		ConnectTimeout:        cfg.ConnectTimeout,
+		HostSelectionPolicy:   hostSelection,
```

Comment:
[client.go](https://github.com/taylanisikdemir/cadence/blob/e0a1d20fc093851f38ad074c06202ea355c6ca72/common/persistence/nosql/nosqlplugin/cassandra/gocql/client.go#L96) needs to be updated to use this value

Reply:
lol, thanks!